src/utils.ts

```diff
@@ -3,12 +3,11 @@ import {
   BigInt,
   ByteArray,
   crypto,
-  Entity,
   EthereumEvent,
   store,
   Value,
 } from '@graphprotocol/graph-ts';
-import {Debug } from './types/schema';
+import { Debug } from './types/schema';
```

Comment:
is this needed ?

Reply:
The space? No, it just annoyed me 🙃

Reply:
The Debug

Reply:
Not exactly necessary, but it's good to keep a debug function as it's sometimes helpful.